app/screens/AppNavigator/screens/Balances/ConvertScreen.tsx

```diff
@@ -0,0 +1,184 @@
+import { NavigationProp, StackActions, useNavigation } from '@react-navigation/native'
+import { StackScreenProps } from '@react-navigation/stack'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useCallback, useState } from 'react'
+import { TouchableOpacity } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import tailwind from 'tailwind-rn'
+import { Text, TextInput, View } from '../../../../components'
+import { PrimaryColor, PrimaryColorStyle } from '../../../../constants/Theme'
+import { useTokensAPI } from '../../../../hooks/wallet/TokensAPI'
+import { BalanceParamList } from './BalancesNavigator'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+import { Ionicons } from '@expo/vector-icons'
+import { PrimaryButton } from '../../../../components/PrimaryButton'
+import { translate } from '../../../../translations'
+import { getWhaleClient } from '../../../../middlewares/api/whale'
+import { getDefaultWallet } from '../../../../middlewares/wallet'
+
+export type ConversionMode = 'utxosToAccount' | 'accountToUtxos'
+type Props = StackScreenProps<BalanceParamList, 'ConvertScreen'>
+
+interface ConversionIO extends AddressToken {
+  unit: 'UTXOS' | 'Token'
+}
+
+export function ConvertScreen (props: Props): JSX.Element {
+  const navigation = useNavigation<NavigationProp<BalanceParamList>>()
+
+  const tokens = useTokensAPI()
+  const [sourceToken, targetToken] = getDFIBalances(props.route.params.mode, tokens)
+
+  const [amount, setAmount] = useState<string>('0')
+  const convAmount = new BigNumber(amount).isNaN() ? '0' : new BigNumber(amount).toString()
+  const resultBal = new BigNumber(targetToken.amount).plus(convAmount)
+  const account = getDefaultWallet().get(0)
+  const whaleApiClient = getWhaleClient()
+
+  const convert = useCallback(() => {
+    constructSignedConversionAndSend(
+      whaleApiClient,
+      account,
+      props.route.params.mode,
+      new BigNumber(amount)
+    ).then(() => {
+      navigation.dispatch(StackActions.popToTop())
+    }).catch(e => console.log(e)) // TODO: display error
+  }, [amount])
+
+  return (
+    <View style={tailwind('w-full h-full')}>
+      <ScrollView style={tailwind('w-full flex-col flex-1 bg-gray-100')}>
+        <ConversionInput
+          current={amount}
+          unit={sourceToken.unit}
+          balance={new BigNumber(sourceToken.amount)}
+          onChange={(val) => {
+            if (sourceToken.unit === 'UTXOS' && new BigNumber(val).eq(sourceToken.amount)) {
+              setAmount(new BigNumber(val).minus(0.0001).toString()) // spare for fee
+            } else {
+              setAmount(val)
+            }
+          }}
+        />
+        <View style={tailwind('w-full justify-center items-center p-2')}>
+          <Ionicons name='arrow-down' size={16} color={PrimaryColor} />
```

Comment:
We should use MaterialIcons. For reference - https://icons.expo.fyi/

Reply:
Interesting, both from same package. They looks the same too, not sure what's the difference.

Reply:
Most of our designs are using MaterialIcons. Also, if I remember correctly, I think Fuxing mentioned that it's cached